lib/alephant/broker/response/base.rb

```diff
@@ -35,7 +35,7 @@ def setup; end
         private
 
         def log_status
-          add_no_cache_headers if status !~ /200/
+          add_no_cache_headers if status != 200
         end
 
```

Comment:
Hi @samfrench I think this change is fine. Using `unless` wouldn't make it any more readable 

Something I would like to see changed though is the name of the method. It suggests we want to log something and then we do nothing of the sort. This method should be `no_cache` or something similar